docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -0,0 +1,109 @@
+.. role:: underline
+    :class: underline
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+  
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+.. _bliss:
+
+==================
+BLISS Style Guide
+==================
+
+Building Language Inclusivity into Simulation Science (BLISS) is an important part of
+our team's work! Here you will find our ongoing style guide for incorporating inclusive 
+language into our day-to-day work.
+
+
+1.0 Overview 
+++++++++++++
+
+1.1 Guiding Principles
+----------------------
+In this section, we will outline the guiding principles of BLISS.
+
+1.2 Glossary of General Terminology
+-----------------------------------
+In this section, we will create a glossary of general terminology that will be used 
+throughout this style guide (e.g., for starters, we should explicitly define sex and 
+gender; SGM).
+
+1.3 Visuals
+-----------
+In this section, we will include some visuals for accessible, quick information 
+(e.g. do's and don't's; what this is and what this isn't).
+
+2.0 Acknowledging Limitations in Peer Research
+++++++++++++++++++++++++++++++++++++++++++++++
+In this section, we will discuss how to acknowledge limitations in external data sources
+used in our research, the different scenarios in which sex/gender conflation may occur and
+how to appropriately articulate these issues in our reports, publications, and other work.  
+
+2.1 Terminology 
+---------------
+
+2.2 Methodology
+---------------
+
+2.3 Imperfect Proxies 
```

Comment:
I think this could go under "Methodology" if that makes sense to people (especially @alibow who I believe is planning to write this section?).